README.md

```diff
@@ -106,40 +106,19 @@ install.packages(
 )
 ```
 
-#### Linux (aarch64) / macOS (aarch64)
-
-GitHub releases also provide pre-compiled packages for arm64 (aarch64)
-platforms.
-
-These are a little different from above. To install, requiring make (and
-Xcode for macOS) but not rustc / cargo. They are actually source
-releases bundled with a pre-compiled object file `./inst/libr_polars.a`.
-The final linking / building of any R packages must be done on the
-native OS/architecture, but that should take only ~20 seconds.
+### Build from source
 
-``` r
-# Linux
-install.packages(
-  "https://github.com/pola-rs/r-polars/releases/latest/download/polars_cross_aarch64-unknown-linux-gnu.tar.gz",
-  repos = NULL
-)
-```
+For source installation, pre-built Rust libraries may be available if
+the environment variable `NOT_CRAN` is set to `"true"`. (Or, set
+`LIBR_POLARS_BUILD` to `"false"`)
```

Comment:
I suppose that setting `NOT_CRAN` to `"true"` could have some side effects if a package developer installs `polars` in the same environment as the package they develop. Maybe we should prefer setting `LIBR_POLARS_BUILD = "false"` instead?

Reply:
Yes, `NOT_CRAN` can have meaning in multiple packages, so we should use `LIBR_POLARS_BUILD` to have it affect only the polars package.

Indeed, downloading binary libraries when `NOT_CRAN=ture` is a behavior borrowed from the `arrow` package.